Client/qtTeamTalk/channel.ui

```diff
@@ -42,6 +42,9 @@
             <property name="text">
              <string notr="true">TextLabel</string>
             </property>
+            <property name="textInteractionFlags">
```

Comment:
I suggest to also define an accessible name for this dialog because it's not really clear to understand what is this text when tabing to them.

Reply:
I've added 2x accessibleName

Reply:
Like this, screenreader only announce the accessible name, accessible name in this case have to be define in cpp file, as I did for file transfer dialog for example :)